pmu_el0_cycle_counter.c

```diff
@@ -243,6 +270,18 @@ pmuctl_ioctl(struct file *f, unsigned int cmd, unsigned long arg)
 		else
 			on_each_cpu(disable_cycle_counter_el0, NULL, 1);
 		break;
+
+	case PMU_IOC_CNTKCTL: /* Enable Physical Timer Control EL0 access */
```

Comment:
Move to pmu_el0_timer_control.c and call from here.

Reply:
Just to confirm... your suggestion is to move all below handling in pmu_el0_timer_control.c  and callthe related function from this IOCTL handler. 

Please confirm.

Reply:
Yes